README.txt

```diff
@@ -24,16 +24,20 @@ Field, Block
 
 -- INSTALLATION --
 
-Normal Drupal module installation, see http://drupal.org/node/70151 for further
-information.
+Normal Drupal module installation, see
+https://www.drupal.org/documentation/install/modules-themes/modules-7
```

Comment:
My commit message is wrong. The old link here isn't broken, but it is wrong. It's to the Drupal 5/6 plugin installation directions.